### ISSUE TITLE
Playing or recording voice message stops any audio

### ISSUE THREAD

**Issue by stillalex** [signalapp/Signal-iOS#3056]
- [x] I have searched open and closed issues for duplicates

----------------------------------------

### Bug description
Interacting with voice messages stops any background playback app (music, podcast). It should pause instead of completely stopping. What I mean is once I'm done listening / recording an audio message whatever I had playing in the background should continue playing.
The Audio Call feature seems to respect this, as far as my limited testing showed: music stops during the call, restarts once the call is done. It would be nice to have the audio messages behave the same way.

I have seen a reference of this in #2100 where stopping the background audio was mentioned. maybe that approach needs another look to take other background apps in consideration.

### Steps to reproduce
- start 'Apple Music' app
- Open Signal (music is still playing in the backgroud)
- tap on a voice message, music stops, recording starts playing
- recording ends

**Actual result:** when the recording ends, music app does not start playing again
**Expected result:** once I'm done listening, music should restart, similar to the 'audio call' feature

### Device info

**Device**: iPhone SE
**iOS version**: iOS 11.2.5
**Signal version:** 2.19.7.6

**Comment by michaelkirk** [comment]
This has already been fixed for our next release 2.20.0, which is currently in beta. 

Interested in being a beta tester? Contact support@signal.org with the subject "Sign me up for the Signal-iOS beta"

Bear in mind that, as a beta release, it's less stable software. That can mean crashing and data loss, not for the feint of heart.

**Comment by stillalex** [comment]
thanks @michaelkirk!
I've sent the email for the beta test.

**Comment by stillalex** [comment]
> Interested in being a beta tester? Contact support@signal.org with the subject "Sign me up for the Signal-iOS beta"
this email address doesn't work, but I managed to find this page [0] mentioning support@whispersystems.org


[0] https://support.signal.org/hc/en-us/articles/115000269532-How-do-I-join-Signal-s-beta-
